recipe/meta.yaml

```diff
@@ -5,6 +5,8 @@
 {% set rpm_url = "http://mirror.centos.org/centos/7.8.2003/os/x86_64/Packages" %}  # [cross_target_platform == "linux-64"]
 {% set rpm_url = "http://mirror.centos.org/altarch/7/os/aarch64/Packages" %}       # [cross_target_platform == "linux-aarch64"]
 {% set rpm_url = "http://mirror.centos.org/altarch/7/os/ppc64le/Packages" %}       # [cross_target_platform == "linux-ppc64le"]
+{% set rpm_url = "http://download.sinenomine.net/clefos/7/os/s390x" %}             # [cross_target_platform == "linux-s390x"]
+{% set rpm_url_clfs_narch = "http://download.sinenomine.net/clefos/7/os/noarch" %} # [cross_target_platform == "linux-s390x"]
```

Comment:
```suggestion
{% set rpm_url_clfs_narch = "http://download.sinenomine.net/clefos/7/os/noarch" %}  # [cross_target_platform == "linux-s390x"]
```

Reply:
@beckermr Thanks for catching .. adding it in next commit.

Reply:
fixed the space issue now and indented accordingly in latest commit. Thanks!